api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -1225,15 +1229,30 @@ private void setBooleanConcepts() {
 			trueConcept = Context.getConceptService().getConcept(
 			    Integer.parseInt(Context.getAdministrationService().getGlobalProperty(
 			        OpenmrsConstants.GLOBAL_PROPERTY_TRUE_CONCEPT)));
+			initializeLazyPropertiesForConcept(trueConcept);
+			
 			falseConcept = Context.getConceptService().getConcept(
 			    Integer.parseInt(Context.getAdministrationService().getGlobalProperty(
 			        OpenmrsConstants.GLOBAL_PROPERTY_FALSE_CONCEPT)));
+			initializeLazyPropertiesForConcept(falseConcept);
 		}
 		catch (NumberFormatException e) {
 			log.warn("Concept ids for boolean concepts should be numbers");
 		}
 	}
-	
+
+	private void initializeLazyPropertiesForConcept(Concept concept) {
+		Hibernate.initialize(concept.getRetiredBy());
+		Hibernate.initialize(concept.getCreator());
+		Hibernate.initialize(concept.getChangedBy());
```

Comment:
You can get the lazily loaded property from Concept.hbm.xml

Reply:
I see in `concept.hbm.xml`, `Creator, ChangedBy and RetiredBy` are not marked as `lazy=true`. However these were throwing the LazyInitException. 
Please have a look at the [screenshot](https://talk.openmrs.org/uploads/default/original/2X/5/51b275ed4e3fb5d27c92cdddcb0b72eca8ea7940.png).